src/renderer/viz/expressions/RampImage.js

```diff
@@ -0,0 +1,88 @@
+import defaultSVGs from '../defaultSVGs';
+import { checkType, checkInstance } from './utils';
+import Property from './basic/property';
+import CategoryIndex from './CategoryIndex';
+import ListImage from './ListImage';
+import SVG from './SVG';
+import Base from './base';
+import { OTHERS_GLSL_VALUE, DEFAULT_OPTIONS, DEFAULT_OTHERS } from './constants';
+
+export default class RampImage extends Base {
+    _bindMetadata (metadata) {
+        Base.prototype._bindMetadata.call(this, metadata);
+        this.type = this.palette.childType;
+
+        if (this.others === DEFAULT_OTHERS) {
+            this.others = new SVG(defaultSVGs.circle);
+        } else {
+            checkType('ramp', 'others', 2, 'image', this.others);
+        }
+
+        if (this.input.isA(Property)) {
+            this.input = new CategoryIndex(this.input);
```

Comment:
Do you think we should move this to `Ramp` _bindMetadata?, this code is duplicated with the one in `RampGeneric`

Reply:
I prefer to keep it separated because RampGeneric does different things, in the RampImage for instance we want to set the default `others` to an SVG, and this is something we need to set in the bindMetadata.